com.tencent.WeChat.yaml

```diff
@@ -0,0 +1,105 @@
+id: com.tencent.WeChat
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: wechat
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  # System tray icon
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-1
```

Comment:
This won't work, if it uses the legacy tray you need `org.kde.*` with an exception to `finish-args-wildcard-kde-own-name` https://docs.flathub.org/docs/for-app-authors/linter#exceptions

Reply:
yes, it uses legacy tray

Reply:
https://github.com/flathub-infra/flatpak-builder-lint/pull/364